tools/src/test/java/org/apache/kafka/tools/reassign/ReassignPartitionsCommandTest.java

```diff
@@ -144,29 +143,6 @@ public void testReassignmentWithAlterPartitionDisabled() throws Exception {
         executeAndVerifyReassignment();
     }
 
-    @ClusterTests({
-        @ClusterTest(types = {Type.KRAFT, Type.CO_KRAFT}, serverProperties = {
-            @ClusterConfigProperty(id = 1, key = INTER_BROKER_PROTOCOL_VERSION_CONFIG, value = "3.3-IV0"),
-            @ClusterConfigProperty(id = 2, key = INTER_BROKER_PROTOCOL_VERSION_CONFIG, value = "3.3-IV0"),
-            @ClusterConfigProperty(id = 3, key = INTER_BROKER_PROTOCOL_VERSION_CONFIG, value = "3.3-IV0"),
-        })
-    })
-    public void testReassignmentCompletionDuringPartialUpgrade() throws Exception {
```

Comment:
Why was this removed?

Reply:
This was a test for some zk mode functionality.
```
-        // Test reassignment during a partial upgrade when some brokers are relying on
-        // `AlterPartition` and some rely on the old notification logic through Zookeeper.
```